lib/grnds/looker/hmac.rb

```diff
@@ -25,6 +25,7 @@ def getUrl()
           json_permissions = @config[:permissions].to_json
           json_models = @config[:models].to_json
           json_access_filters = @config[:filters].to_json
+          json_user_attributes = @config[:user_attributes].to_json
```

Comment:
How are these set?

Reply:
Oh. The caller is responsible for setting them, thus the `black` PR. Thanks, @ajuvekar .